src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/domain/entities/roleassignment/RoleName.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi.domain.entities.roleassignment;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum RoleName {
+    TRIBUNAL_CASEWORKER("tribunal-caseworker");
```

Comment:
Will it be worth to also have `senior-tribunal-caseworker` to avoid the "failing to parse" in case there is a role with that role name.

Reply:
done